### ISSUE TITLE
Spotted some split vote data inconsistencies

### ISSUE THREAD

**Issue by BMarkham06** [doehm/survivoR#19]
Hi! Love the data set, thanks for maintaining it! 

I noticed a few inconsistencies with the split vote data in the vote_history file that I was hoping could be corrected. Most of these are cases where a castaway is listed as voting for one castaway but then is also listed as part of a split vote for two different castaways. Another is just a typo. Note that I haven't seen any of the seasons in question myself, so I'm not sure which one is incorrect in each case:

- Season 16, episode 2: Tracy's name is misspelled as "Tracey" when referencing her in the split_vote column
- Season 26, episode 10: Reynold is listed as voting for Phillip in the vote column, but also listed as part of a split vote on Eddie and Malcolm in the split_vote column
- Season 26, episode 12: Andrea and Erik are listed as receiving split votes in the split_vote column, but the players listed as part of the this split vote are all listed as voting for Andrea or Eddie in the vote column
- Season 31, episode 11: Abi-Maria is listed as voting for Stephen in the vote column but is listed as part of a Abi-Maria, Joe split in the split_vote column
- Season 42, episode 12: Omar is listed as voting for Jonathon in the vote column but is listed as part of a Romeo, Omar split in the split_vote column.

Thank again, I appreciate it. 💯

**Comment by doehm** [comment]
Hey, thanks! Much appreciated. It's always great to hear from people using the data.

Thanks for sending the corrections in, particularly about the split_vote field. It's one of the more nuanced fields. I've made those corrections and pushed to git. Not sure how those slipped in but I've written a test to prevent them in the future.

I'm going to rewatch S42E12. I've also listed Romeo as part of the split between Romeo and Omar. It's possible for that to be the case and I probably had my reasons but still best to check. I can't quite remember the exact details. I'll let you know if I make further updates to split_vote.

Cheers!
